index.html

```diff
@@ -123,7 +123,7 @@ <h2>Multi-Post
 
         <section class="project-section">
             <figure>
-                <img src="./Images/gym_fit.png" alt="" id="snap-shot-image">
+                <img src="./Images/gym_fit.png" alt="" class="snap-shot-image">
             </figure>
```

Comment:
- [ ] It will be better 😊 if you put the sections inside a container to align the content like the Figma Design as you see here the distance between the section from the both left and right sides is not like the Figma and it is needed to increase to be like the Figma Design 👌🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/235675764-6b392125-dc6f-4325-9719-25a09b7b6f80.png)   |  ![image](https://user-images.githubusercontent.com/83360636/235675688-9e5e0bfd-5df4-4063-b779-fc7f76539290.png)  |